contracts/OpenSTValue.sol

```diff
@@ -73,9 +72,10 @@ contract OpenSTValue is OpsManaged, Hasher {
     uint256 private constant BLOCKS_TO_WAIT_LONG = 80667;
     // ~1hour, assuming ~15s per block
     uint256 private constant BLOCKS_TO_WAIT_SHORT = 240;
-
-
-    /// register the active stakes and unstakes
+    
+    // storage for staking intent hash of active staking intents
+    mapping(bytes32 /* intentHash */ => bytes32) public intents;
```

Comment:
We need to define index position of intents mapping in OpenSTUtility contract.
There needs to be unit test cases for the index position.

Index position of intents mapping is needed to generate storage proof.

Reply:
Yes, this was recommended by @benjaminbollen to be made into a separate ticket IIRC. I can pick up it up.